package.json

```diff
@@ -11,14 +11,19 @@
 	],
 	"license": "agpl",
 	"private": true,
+	"files": [
```

Comment:
also, we need to use [`.npmignore`](https://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html) to prevent non-intentional exports like `/js` folder

Reply:
I think for now we are fine as we explicitly list the dirs we want to include in the files list. I think we'd need .npmignore if we wanted to ignore files from within those dirs.